src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```
@@ -43,6 +43,8 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
     <EmbedAssembliesIntoApk Condition=" '$(EmbedAssembliesIntoApk)' == '' ">true</EmbedAssembliesIntoApk>
     <AndroidManagedSymbols Condition=" '$(AndroidManagedSymbols)' == '' ">true</AndroidManagedSymbols>
+    <AndroidPackageFormat Condition=" '$(AndroidPackageFormat)' == '' " >aab</AndroidPackageFormat>
+    <AndroidAdditionalPackageFormats Condition=" '$(AndroidAdditionalPackageFormats)' == '' " >apk</AndroidAdditionalPackageFormats>
```

Should this maybe also check:
```suggestion
    <AndroidAdditionalPackageFormats Condition=" '$(AndroidPackageFormat)' == 'aab' and '$(AndroidAdditionalPackageFormats)' == '' " >apk</AndroidAdditionalPackageFormats>
```